### PR TITLE
Documentation: teaching: labs: block_device_drivers: Update exercises

### PR DESCRIPTION
**Pull request by valighita**
Signed-off-by: Valentin Ghita valx92@gmail.com

### CODE REVIEW

File: Documentation/teaching/labs/block_device_drivers.rst [hunk 1]
```diff
@@ -921,7 +921,7 @@ Follow the comments marked with **TODO 2**. Use the
 
 .. hint:: Review the `Register a disk`_ and `Process a request`_ sections.
 
-Fill in the :c:func:`my_block_request` function which processes the request queue
+Fill in the :c:func:`my_block_request` function to processes the request queue
```
Comment (teoserbanescu) [inline Documentation/teaching/labs/block_device_drivers.rst]: s/processes/process/g

File: Documentation/teaching/labs/block_device_drivers.rst [hunk 2]
```diff
@@ -949,27 +949,27 @@ To generate writing requests, use the command:
 
    echo "abc"> /dev/myblock
 
-Notice that is created a write request preceded by a read request. The read
-request takes place to read the block from the disk and "update" in its content
-what it was provided by the user without overwriting the rest. After reading and
+Notice that a write request is created preceded by a read request. The request
```
Comment (razvand) [inline Documentation/teaching/labs/block_device_drivers.rst:952]: s/created preceded/preceded/g

File: Documentation/teaching/labs/block_device_drivers.rst [hunk 3]
```diff
@@ -982,26 +982,23 @@ information, use the fields of the :c:type:`struct request` structure.
...
+virtual machine at ``make copy`` and can be run on the QEMU virtual machine
+using the command:
 
 .. code-block:: shell
 
    ./ram-disk-test
 
 There is no need to insert the module into the kernel, it will be inserted by
-the :command:`ram-disk-test` executable.
+the test command.
```
Comment (razvand) [inline Documentation/teaching/labs/block_device_drivers.rst:999]: Is it `test` or is it `ram-disk-test`? It should be forrmatted with backticks

File: Documentation/teaching/labs/block_device_drivers.rst [hunk 4]
```diff
@@ -1139,14 +1140,7 @@ the pages of each :c:type:`struct bio` structure and access its associated
...
 
-For testing, use the :file:`ram-disk-test.c` test file. Compile it on
-the host, using the command:
-
-.. code-block:: shell
-
-    make -f Makefile.test
-
-and then run it in the QEMU virtual machine using the following command:
+For testing, use the :file:`ram-disk-test.c` test file:
```
Comment (razvand) [inline Documentation/teaching/labs/block_device_drivers.rst:1143]: Why did you remove the instructions to compile the test file? Are they presented earlier?

Comment (valighita) [reply Documentation/teaching/labs/block_device_drivers.rst:1143]: Yes, they are described for exercise 3.
